target_chains/ethereum/contracts/contracts/executor/Executor.sol

```diff
@@ -87,7 +89,13 @@ contract Executor {
         if (len == 0) revert ExecutorErrors.InvalidContractTarget();
 
         bool success;
-        (success, response) = address(callAddress).call(gi.callData);
+        if (gi.value > 0) {
```

Comment:
let me guess: calling with zero value is an error? If so, can you add a comment explaining why this if statement is necessary?